.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
The `.env` file should also be in the `.dockerignore`. It contains the API's  credentials.

You can probably use the `.gitignore`.